src/ImageMorphology.jl

```diff
@@ -68,8 +69,8 @@ export
     #geodesy.jl
     underbuild,
     overbuild,
-    hmaxima,
-    hminima,
+    hmaximum,
```

Comment:
The correct theoric terms is h_maxima :) 
https://en.wikipedia.org/wiki/H-maxima_transform

Reply:
Ha, yes I was aware of it. I thought it might be a good idea to use `hmax`/`hmin` or `hmaximum`/`hminimum` to make them look similar to the Base functions `max`/`min` and `maximum`/`minimum`.

Reply:
Aside question may you could give me a link to a good julia Linter 
So i could format properly :( and you don't have to worry on this

Reply:
I think someone from the Mathematical Community or follow courses on it, will try to find something like
hmaxima or h_maxima

Reply:
> Aside question may you could give me a link to a good julia Linter
So i could format properly :( and you don't have to worry on this

Actually I do manual format in VSCode, for instance, 

![image](https://user-images.githubusercontent.com/8684355/163954435-2de3bb2d-2f0b-4282-8e41-574c16b40015.png)

This feature is currently backed by https://github.com/domluna/JuliaFormatter.jl